src/fu-debug.c

```diff
@@ -107,6 +88,35 @@ fu_debug_handler_cb (const gchar *log_domain,
 	}
 }
 
+static void
+fu_debug_ignore_cb (const gchar *log_domain,
+		    GLogLevelFlags log_level,
+		    const gchar *message,
+		    gpointer user_data)
+{
+	FuDebug *self = (FuDebug *) user_data;
+
+	/* refresh to verbose logging */
+	if (self->verbose ||
+	    g_strcmp0 (g_getenv ("FWUPD_VERBOSE"), "1") == 0) {
```

Comment:
I'm not sure why we need the environment variable _and_ the `verbose` static variable...

Reply:
I removed the use of the static variable.